_api-reference/index-apis/shrink-index.md

```diff
@@ -56,6 +55,6 @@ timeout | Time | How long to wait for the request to return a response. Default
 You can use the request body to configure some index settings for the target index. All fields are optional.
 
 Field | Type | Description
-alias | Object | Sets an alias for the target index. Can have the fields `filter`, `index_routing`, `is_hidden`, `is_write_index`, `routing`, and `search_routing`. See [index aliases]({{site.url}}{{site.baseurl}}/opensearch/rest-api/alias/#request-body).
-settings | Object | Index settings you can apply to your target index. See [index settings]({{site.url}}{{site.baseurl}}/opensearch/rest-api/index-apis/create-index/#index-settings).
+alias | Object | Sets an alias for the target index. Can have the fields `filter`, `index_routing`, `is_hidden`, `is_write_index`, `routing`, and `search_routing`. See [index aliases]({{site.url}}{{site.baseurl}}/api-reference/alias/#request-body).
```

Comment:
Current link title "index aliases" doesn't match anything on the page. It looks like the intent here was to link to the `Alias` API page. If so -

`[Alias]({{site.url}}{{site.baseurl}}/api-reference/alias)`